tidb-cloud/changefeed-sink-to-apache-kafka.md

```diff
@@ -31,29 +31,106 @@ If your Apache Kafka service is in an AWS VPC that has no internet access, take
+
+        For changelogs of non-row events, such as Create Schema Event, you can specify a topic name in the **Default Topic Name** field. The changefeed will create a topic accordingly to collect such changelogs.
+
+   - **Distribute changelogs by database to Kafka Topics**
+
+        If you want the changefeed to create a dedicated Kafka topic for each database, choose this mode. Then, all Kafka messages of a database are sent to a dedicated Kafka topic. You can define topic names of databases by setting the topic prefix, separator, and suffix. For example, if you set the separator as `_`, the topic names are in the format of `<Prefix>_<DatabaseName>_<Suffix>`.
+
+   - **Send all changelogs to one specified Kafka Topic**
+
+        If you want the changefeed to create one Kafka topic for all changelogs, choose this mode. Then, all Kafka messages of the cluster will be sent to one Kafka topic. You can define the topic name in the **Default Topic Name** field.
+
+5. In the **Partition Distribution** area, you can decide which partition a Kafka message will be sent to:
+
+   - **Distribute changelogs by index value to Kafka partition**
+
+        If you want the changefeed to send Kafka messages of a table to different partitions, choose this distribution method. The index value of a row changelog will determine which partition the changelog is sent to. This distribution method provides a better partition balance and ensures row-level orderliness.
+
+   - **Distribute changelogs by table to Kafka partitions**
+
+        If you want the changefeed to send Kafka messages of a table to one Kafka partition, choose this distribution method. The table name of a row changelog will determine which partition the changelog is sent to. This distribution method ensures table orderliness but might cause unbalanced partitions.
+
+6. In the **Topic Configuration** area, configure the following numbers. The changefeed will automatically create the Kafka topics according to the numbers.
+
+   - **Replication Factor**: controls how many Kafka servers each Kafka message is replicated to.
+   - **Partition Number**: controls how many partitions exist in a topic.
+
+7. Click **Next** to check the configurations you set and go to the next page.
+
+## Step 4. Review the configurations
 
-You must prepare a Topic before creating an Apache Kafka Sink. Based on table, the Sink will distribute data to different partitions of the Topic.
+On this page, you can review all the changefeed configurations that you set.
 
-## Create a Sink
+If you find any error, you can go back to fix the error. If there is no error, you can click the check box at the bottom, and then click **Create** to create the changefeed.
 
-After completing the Prerequisites, you can sink your data to Apache Kafka.
+## Step 5. Manage the changefeed
```

Comment:
This section includes changefeed operations, not a necessary step of the whole procedure. Better make it an independent section. 
```suggestion
## Manage the changefeed
```